charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/blackbox-exporter.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: service-check
+  namespace: kube-system
+  labels:
+    garden.sapcloud.io/role: system-component
+    component: service-check
+    addonmanager.kubernetes.io/mode: Reconcile
+    origin: gardener
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
+  selector:
+    matchLabels:
+      component: service-check
+  template:
+    metadata:
+      labels:
+        origin: gardener
+        garden.sapcloud.io/role: system-component
+        component: service-check
+    spec:
+      containers:
+      - name: blackbox-exporter
+        image: {{ index .Values.images "blackbox-exporter" }}
+        args:
+        - --config.file=/etc/blackbox_exporter/blackbox.yaml
+        imagePullPolicy: IfNotPresent
+        # TODO: network rootnamespace
+        securityContext:
+          privileged: true
```

Comment:
This is not needed?
Only `ICMP` probe requires extra privileges. https://github.com/prometheus/blackbox_exporter/blob/22be14377a74e81e6c4c77a5577c14f944aa2277/README.md#permissions

Reply:
I removed this.